src/wolfgame.py

```diff
@@ -979,16 +979,29 @@ def fjoin(var, wrapper, message):
         "join_deadchat": join_deadchat,
         "vote_gamemode": vote_gamemode
         })
+   
     if not evt.dispatch(var, wrapper, message, forced=True):
         return
     noticed = False
     fake = False
     if not message.strip():
         evt.data["join_player"](var, wrapper, forced=True)
-
-    for tojoin in re.split(" +", message):
+   
+    parts = re.split(" +", message)
+    possible_users = {u.lower().nick for u in wrapper.target.users}
+    if not botconfig.DEBUG_MODE:
+        match = complete_one_match(users.lower(parts[0]), possible_users)
+        if match:
+            tojoin = [match]
```

Comment:
Use a different variable name, even make it something like `to_join`, because I don't want the loop variable further down to be the same as the iterable.